doc/cite.rst

```diff
@@ -1,17 +0,0 @@
-.. _cite:
-
-How to cite MNE
----------------
-
-If you use the implementations provided by the MNE software in your research, you should cite:
-
-    [1] A. Gramfort, M. Luessi, E. Larson, D. Engemann, D. Strohmeier, C. Brodbeck, L. Parkkonen, M. Hämäläinen, `MNE software for processing MEG and EEG data <http://www.ncbi.nlm.nih.gov/pubmed/24161808>`_, NeuroImage, Volume 86, 1 February 2014, Pages 446-460, ISSN 1053-8119, `[DOI] <http://dx.doi.org/10.1016/j.neuroimage.2013.10.027>`__
-
-If you use the Python code you should cite as well:
-
-    [2] A. Gramfort, M. Luessi, E. Larson, D. Engemann, D. Strohmeier, C. Brodbeck, R. Goj, M. Jas, T. Brooks, L. Parkkonen, M. Hämäläinen, `MEG and EEG data analysis with MNE-Python <http://journal.frontiersin.org/article/10.3389/fnins.2013.00267/abstract>`_, Frontiers in Neuroscience, Volume 7, 2013, ISSN 1662-453X, `[DOI] <http://dx.doi.org/10.3389/fnins.2013.00267>`__
-
-To cite specific versions of the software, you can use the DOIs provided by
-`Zenodo <https://zenodo.org/search?ln=en&p=mne-python>`_.
-
-You should as well cite the related method papers, some of which are listed in :ref:`ch_reading`.
```

Comment:
well I think it's easier to find if you keep it in it's own page so we can access it from the font page not just in a subsection of FAQ.